### ISSUE TITLE
Ability for !nuke command to timeout users that type the same nuked phrase a period of time after nuke was issued.

### ISSUE THREAD

**Issue by Mr0ll3** [pajbot/pajbot2#519]
Currently when !nuke is used some users that was not target by the nuke, is typing the same phrase that was nuked which leads to moderators having to either manually time those users out or use the same nuke again.
It would make moderation less of a hassle in these situations if the bot took care of these users that start using the nuked phrase after other uses has been punished.

To clarify, after !nuke has been issued, a grace period (lets say 30 seconds) is set where users that type the same phrase will also be timed out for same period that was set in the nuke.

Command in question:
[https://nymn.pajbot.com/commands/1011-nukeprefix](https://nymn.pajbot.com/commands/1011-nukeprefix)

**Comment by ALazyMeme** [comment]
Correct me if I'm wrong, but I believe that's the radiation option on fossabot?
CC: @Felanbird

**Comment by Felanbird** [comment]
> Correct me if I'm wrong, but I believe that's the radiation option on fossabot?
> CC: @Felanbird

Yes that's the same premise.

**Comment by Mr0ll3** [comment]
> Correct me if I'm wrong, but I believe that's the radiation option on fossabot?
> CC: @Felanbird

Yeah I checked if there was any documentation for it that I could reference, but the documentation for fossabot is pretty limited.
The nuke command can also only be added / used if Aiden whitelist it for said channel.

**Comment by Felanbird** [comment]
> Yeah I checked if there was any documentation for it that I could reference, but the documentation for fossabot is pretty limited.
> The nuke command can also only be added / used if Aiden whitelist it for said channel.

Yes nuke is an undocumented command on Fossabot, but I don't believe it was ever whitelist required.

**Comment by Mr0ll3** [comment]
> > Yeah I checked if there was any documentation for it that I could reference, but the documentation for fossabot is pretty limited.
> > The nuke command can also only be added / used if Aiden whitelist it for said channel.
> 
> Yes nuke is an undocumented command on Fossabot, but I don't believe it was ever whitelist required.

I was reading it wrong when I quickly looked at one of Aidens comments. He stated it's a private command on the bot and he whitelist the bot on channels after they contact him.

>To access the nuke function it's a private command on Fossabot, contact me for access to that since I whitelist channels.

https://www.reddit.com/r/LivestreamFail/comments/dvxw0s/mass_bot_ban_on_xqcs_stream/f7h05km/

**Comment by Felanbird** [comment]
> > To access the nuke function it's a private command on Fossabot, contact me for access to that since I whitelist channels.
> 
> https://www.reddit.com/r/LivestreamFail/comments/dvxw0s/mass_bot_ban_on_xqcs_stream/f7h05km/

This was the usage of `!filesay`, not `!nuke`. The wording is just confusing as he is stating the means to "nuke" a large amount of bots. 
> He sent me a huge `list` of 4000 bots that just got made

While the suggestion of needing whitelist may have been relevant at the time, it was later realized that the `!filesay` tool was too important to not be available globally.
